domparsing/XMLSerializer-serializeToString.html

```diff
@@ -145,8 +152,6 @@ <h1>domparsing_XMLSerializer_serializeToString</h1>
 
 test(function() {
   assert_equals(serialize(parse('<root><child/></root>')), '<root><child/></root>');
-  assert_equals(serialize(parse('<root><child xmlns=""/></root>')), '<root><child/></root>');
```

Comment:
It seems we lost this particular input @tkent-google. A root without xmlns declaration and a child with one. Could you restore that?

Reply:
Sure!